src/ui/component/dayPicker/dayPicker.tsx

```diff
@@ -0,0 +1,60 @@
+import { type FC, useMemo } from 'react'
+import type { SelectSingleEventHandler } from 'react-day-picker'
+import { DayPicker as RDayPicker } from 'react-day-picker'
+import { enUS, fr, de } from 'date-fns/locale'
+import i18n from '@/ui/i18n'
+import 'react-day-picker/dist/style.css'
+import './dayPicker.scss'
+
+const localeMap = new Map([
+  ['en', enUS],
+  ['fr', fr],
+  ['de', de]
+])
+
+type DayPickerProps = {
+  onSelect: SelectSingleEventHandler
+  fromYear: number
+  toYear: number
+  fromDate?: Date
+  toDate?: Date
+  selected?: Date
+  title?: string
+  isoWeek?: boolean
+  showOutsideDays?: boolean
+}
+
+export const DayPicker: FC<DayPickerProps> = ({
+  onSelect,
+  fromYear,
+  toYear,
+  fromDate,
+  toDate,
+  selected,
+  title,
+  isoWeek = true,
+  showOutsideDays = true
+}) => {
+  const locale = i18n.language
```

Comment:
```suggestion
  const { i18n } = useTranslation()
  const locale = i18n.language
```

Reply:
You are recommending to always using the hook in the context of a component, right?

Reply:
Yes, I think it's the best practice. Since there's the hook available, it's better than importing the `i18n` instance from the utils. That's what you've done in your PR [feat/datepicker-component](https://github.com/okp4/dataverse-portal/blob/fca8b7cba856bbafb4bdb92b81ddbc3870253c2d/src/ui/component/dateInput/dateInput.tsx#L11) and I agree with that. By the way, I'd suggest applying this rule by adding an eslint import rule...

Reply:
Ooops my fault 😬 🤭